ja2r.gemspec

```
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir['{app,config,db,lib}/**/*', 'LICENSE', 'README.md']
 
-  s.add_dependency 'activesupport', '>= 6.1', '< 8.0'
+  s.add_dependency 'activesupport', '>= 6.1', '< 9.0'
```

Nit: Personally I would keep it more strict:


```suggestion
  s.add_dependency 'activesupport', '>= 6.1', '< 8.1'
```

But up to @mkon ;)

---

Yes please set 8.1 as upper bound.

---

@mkon changed to `8.1`. Please have a look. Thanks